### PR TITLE
Deploy in WeChat

### PR DESCRIPTION
**Pull request by abhishek-iiit**
# Deploy in WeChat
Screenshot from 2021-07-22 17-12-58
Screenshot from 2021-07-22 17-13-10
Screenshot from 2021-07-22 17-13-23

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Linked to issue #896

File: docusaurus/docs/howto/deploy-wechat.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Deploy in WeChat
+---
+
+With all-in-one system and massive audience base, deploying WeChaty in WeChat can prove to be a good choice.In this how to guide, you will learn how to deploy the bot in WeChat locally.
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat.md]: The introduction is good. You added the benefit of following this guide and what the users would learn, but it's better if you could explain more, which I meant to mention that we will use starter template as an example in here.

File: docusaurus/docs/howto/deploy-wechat.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+![Deploy in WeChat](../../static/img/docs/howto/IM_platform/deploy-wechat.webp)
+
+## Requirements
+
+1. Your system must have [Node.js](https://nodejs.org/en/download/package-manager/) installed (version >= 12).
+2. Your system must have [Wechaty](https://github.com/wechaty/wechaty) (version >= 0.40).
+3. You must be familiar with [Wechaty Plugins](https://www.npmjs.com/package/wechaty-plugin-contrib).
+
+## Deployment
+
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat.md]: Run `wechaty-puppet-wechat`:

File: docusaurus/docs/howto/deploy-wechat.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+Let's take up an example on how to integrate bot from [starter templete](https://github.com/wechaty/wechaty-getting-started) to WeChat.The steps are similar for all other bots as well.
+
+### Prerequisite
+
+1. Offical Wechaty package: [package/wechaty](https://www.npmjs.com/package/wechaty).
+2. QRCode terminal edition: [package/qrcode-terminal](https://www.npmjs.com/package/qrcode-terminal).
+
+You can follow up the steps mentioned below:
+
+<ol><li> Initialise the project by creating a new folder `my-bot`.</li>
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat.md]: Initialize <- use the American spelling instead.

File: docusaurus/docs/howto/deploy-wechat.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+set WECHATY_PUPPET=wechaty-puppet-wechat
+npm start
+```
+
+</TabItem>
+</Tabs>
+
+## Example
+
+Let's take up an example on how to integrate bot from [starter templete](https://github.com/wechaty/wechaty-getting-started) to WeChat.The steps are similar for all other bots as well.
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat.md]: Add a step between two sentences.

File: docusaurus/docs/howto/deploy-wechat.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+```sh
+npm install wechaty-puppet-wechat
+set WECHATY_PUPPET=wechaty-puppet-wechat
+npm start
+```
+
+</TabItem>
+</Tabs>
+
+## Example
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat.md]: I'm not sure for this title. How do you feel if we use "integrate a bot from starter template to WeChat" ? I'll let you decide for this one. I can't find a great example.

**Review by proudofsimin — CHANGES_REQUESTED** [review]
Just wondering if there are going to be guides for WhatsApp or others.

**Comment by abhishek-iiit** [comment]
Changes done as suggested :)

**Comment by abhishek-iiit** [comment]
Changes done as suggested :)

File: docusaurus/docs/howto/deploy-wechat.md (hunk 2 above)
Comment (proudofsimin) [reply docusaurus/docs/howto/deploy-wechat.md]: Why don't you add this sentence in?

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/howto/deploy-wechat.md (hunk 2 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/deploy-wechat.md]: Added

**Review by proudofsimin — APPROVED** [review]

**Review by vasvi-sood — APPROVED** [review]
This PR is great. LGTM

**Review by lijiarui — DISMISSED** [review]
image

It seems you didn't put this in the nav bar. Please add this to nav bar and then I will review.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by abhishek-iiit** [comment]
merging PR with no conflicts and 4 approvals :)

**Pull request merged**